### PR TITLE
Change title

### PR DESCRIPTION
**Pull request by Markusdreyer**

### CODE REVIEW

File: login-config.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# API documentation configuration file
+# Used for https://vipps.no/developer-pages
+# This file will be fetched by the user's browser
+# Do not store personal or confidental information in this file
+​
```
Comment (krisztianzsobrak) [inline login-config.yaml]: There is an invisible character in each of the empty lines. Maybe it is your IDE that put it there, but the `js-yaml` module cannot decode it this way. Could you check please?

File: getting-started-config.yaml [hunk 2]
```diff
@@ -4,13 +4,13 @@
 # Do not store personal or confidental information in this file
```
Comment (krisztianzsobrak) [inline getting-started-config.yaml:4]: Next time please branch out a new branch from develop.

**Review by krisztianzsobrak — CHANGES_REQUESTED** [review]

File: login-config.yaml (hunk 1 above)
Comment (Markusdreyer) [reply login-config.yaml]: Found the issue, committing now.

File: login-config.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - title: Documentation
+    slug: documentation
+    description: Read the documentation of our login API
+    markdownUrl: vipps-login-api/master/vipps-login-api.md
+    githubUrl: https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api.md
+  - title: FAQ
+    slug: faq
+    description: Questions and answers about the Login API
+    markdownUrl:  vipps-login-api/master/vipps-login-api-faq.md
+    githubUrl: https://github.com/vippsas/vipps-login-api/blob/master/vipps-login-api-faq.md
```
Comment (krisztianzsobrak) [inline login-config.yaml]: This file doesn't exists. Possibly `vipps-login-faq.md`

**Review by krisztianzsobrak — APPROVED** [review]
Well done :rocket:

**Pull request merged**